.github/workflows/lint.yml

```diff
@@ -2,15 +2,11 @@
 ---
 name: CI lint
 
+# API Changes (godoc) and JIRA validation only apply to PRs.
+# Branches are not important, supports chaining PRs.
+
 on:
   pull_request:
-    branches:
```

Comment:
this is still valid, right? 
run the jira lint only when PR is raised against master/release-* branches?